### ISSUE TITLE
Review Documentation for SOPs

### ISSUE THREAD

**Issue by Wkt8** [ebi-ait/hca-ebi-wrangler-central#608]
A bunch of our documentation has slowly gone out of date, with particular examples being: 
- secondary review process (https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/secondary_review_SOP.html) 
- visual index diagram (https://ebi-ait.github.io/hca-ebi-wrangler-central/)

We should take time to go through the entire Hca-ebi-wrangler-central documentation page, and ensure that each document is up to date. If it is not up to date, we should update it. 

**Acceptance criteria for the task:**
- list of documentation created
- all items on the list are 'up-to-date' to a reasonable degree, i.e. could be followed by another wrangler, does not have dead links (dataset tracking sheet)

Current Tasks:
- [x] Dataset suitability
- [x] **Onboarding guide**
- [x] zenhub lanes #746

- [x] add file source for sequence file to hca template!
- [x] review GDPR instructions
- [x] Secondary review guide
- [x] Review account creation for contributors

**Comment by ipediez** [comment]
I'll post here some things I found to be outdated/wrong while reading through the SOPs during the onboarding and wrangling process:

### [HCA Metadata spreadsheet guide for Wranglers](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/HCA_Metadata_Spreadsheet_Guide_Wranglers.md)

- The link with the instructions about how to build the spreadsheet in [this section](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/HCA_Metadata_Spreadsheet_Guide_Wranglers.md#making-the-sheet) is not working.
- All the images in the document are not available
- In [this section](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/HCA_Metadata_Spreadsheet_Guide_Wranglers.md#ontology-columns), the links to HCAO are broken

### [Contributor communication SOP](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/contributor_communication_SOP.md)

- The 'template email' link at 'final confirmation before submission' redirects to the same section, not showing the corresponding template email. This is due to the name of the 'final confirmation before submission' template email section, which is the same as the section in which the template email is linked.
- The first link at 'Follow-up contact' is broken. It redirects to the section 'terms-and-conditions-and-consent', when it should be 'terms-and-conditions'.

### [Dataset wrangling SOP](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/dataset_wrangling_SOP.md)

- The [section about published datasets](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/dataset_wrangling_SOP.md#for-published-datasets-only) includes a link to the spreadsheet generator that currently we don't use. It's not deprecated, it can be used, but the wranglers now directly do it form ingest. Although we can go through this at the onboarding session about ingest, it could be nice to have a section at the SOP about this.
- The link to 'Request ontoly terms' at the [ontology curation section](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/dataset_wrangling_SOP.md#ontology-curation) is not working.
- Add link to [assay cheat sheet]( https://docs.google.com/spreadsheets/d/1H9i1BK-VOXtMgGVv8LJZZZ9rbTG4XCQTBRxErdqMvWk/edit#gid=577259307):
     
### [HCA data portal](https://data.humancellatlas.org/)

- The metadata design documentation link at the [metadata section](https://data.humancellatlas.org/metadata) is not working

### [Dataset suitability](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/dataset_suitability_SOP.md)

- "Contain at least some samples derived from healthy human donors" is wrong, we accept datasets without healthy donors

### Curation of auto-imports SOP

-  Add [ticket 489](https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/489) as a new SOP

### Onboarding documentation

**EBI wrangler onboarding**

- metadata-updates HCA  GitHub team doesn't exist right now, so this should be changed
- the newcomers need to be added though to the metadata schema repo
- The link of bios and photos at miscellanea is broken

[**Timesheet guidance**
](https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=HCA&title=Timesheet+Guidance)

- The webpage is outdated

Spreadsheets

- Use the up-to-date spreadsheet generator: https://github.com/ebi-ait/geo_to_hca/tree/master/docs

**Comment by Wkt8** [comment]
If it is a straightforward change (e.g. image is broken, link is broken, typos), fix it and merge it. It does not require a pull request. Thank you!

**Comment by ipediez** [comment]
hca-ebi-wrangler-central repo documentation list

## SOPs

- [ ] GDPR_Guidelines.md
- [ ] HCA_Metadata_Spreadsheet_Guide_Wranglers.md
- [ ] NCBI_SRA_cold_storage_how_to.md
- [ ] SOPs.md
- [ ] archiving_SOP.md
- [ ] cloud_cost_SOP.md
- [ ] contributor_communication_SOP.md
- [ ] contributor_matrix_addition_SOP.md
- [ ] data_retraction_SOP.md
- [ ] dataset_acknowledgement_SOP.md
- [ ] dataset_suitability_SOP.md
- [ ] dataset_triage_SOP.md
- [ ] dataset_wrangling_SOP.md
- [ ] geo_to_hca_SOP.md
- [ ] hca_to_scea_tools_SOP.md
- [ ] schema_update_prioritisation_SOP.md
- [ ] secondary_review_SOP.md
- [ ] update_a_project_SOP.md
- [ ] update_ena_runs_SOP.md
- [ ] updating_zooma.md

## tools

- [ ] handy_snippets.md
- [ ] setup_public_folder_own_cloud.md
- [ ] tools.md
- [ ] wrangler_tool_survey.adoc

**Comment by idazucchi** [comment]
Enrique already integrated ticket #489 in the [dataset acknowledgement SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_acknowledgement_SOP.html#manual-curation-after-addition)

**Comment by ipediez** [comment]
True, I'm gonna remove that from the list above

**Comment by ipediez** [comment]
https://github.com/ebi-ait/hca-ebi-wrangler-central/pull/640

**Comment by yusra-haider** [comment]
PR ready for review. Waiting on review from wranglers

cc: @ESapenaVentura @ami-day

**Comment by ami-day** [comment]
Done, my feedback is in the pull request.

**Comment by idazucchi** [comment]
@ami-day to review changes to the PR

**Comment by ami-day** [comment]
> #640

I have already done this - i approve of the changes.

**Comment by idazucchi** [comment]
this is also linked to tickets #746 #541

**Comment by idazucchi** [comment]
[PR](https://github.com/ebi-ait/hca-ebi-wrangler-central/pull/900) for wrangling guide and spreadsheet guide

**Comment by idazucchi** [comment]
Remaining things to update - usefule for new joiners
- Dataset suitability
- Onboarding guide
- zenhub lanes #746
- review GDPR instructions

**Comment by idazucchi** [comment]
[PR](https://github.com/ebi-ait/hca-ebi-wrangler-central/pull/934) for the eligibility criteria

**Comment by gabsie** [comment]
@gabsieto see this as well.

**Comment by idazucchi** [comment]
add file source for sequence file to hca template!

**Comment by Wkt8** [comment]
To discuss secondary review and GDPR and onboarding in the documentation hour on Monday.

**Comment by idazucchi** [comment]
PR for the onboarding guide [here](https://github.com/ebi-ait/hca-ebi-wrangler-central/pull/970)

**Comment by Wkt8** [comment]
Waiting for reviews from wranglers

**Comment by MightyAx** [comment]
review comments made!
- [ ] @idazucchi to make changes based on review.

**Comment by idazucchi** [comment]
PR for secondary review and account creation [here](https://app.zenhub.com/workspace/o/ebi-ait/hca-ebi-wrangler-central/issues/984)

**Comment by idazucchi** [comment]
I merged the last PR and I'm moving the ticket to done!
